pontoon/base/static/css/translate.css

```diff
@@ -786,6 +786,13 @@ body > header aside p {
   padding-left: 2%;
 }
 
+#entitylist > .wrapper > ul > li p > span .searchHighlight {
+  color: #7BC876;
```

Comment:
Maybe `#FED271` (Pontoon yellow) instead? Usually yellow color is used for hihglighting. Also, we use `#7BC876` in many other places and it has a special meaning ("translated").